packages/calcite-components/src/components/meter/usage/Advanced.md

```diff
@@ -0,0 +1,16 @@
+A complex meter component.
```

Comment:
Not critical for this PR and more of a doc suggestion, but it might be good to describe what makes it complex or the use case it's representing. cc @geospatialem

Reply:
Updated